lapis/simulator.py

```diff
@@ -6,9 +6,9 @@
 
 from lapis.drone import Drone
 from lapis.job import job_to_queue_scheduler
-from lapis.utility.monitor import Monitoring, collect_pool_statistics, \
-    collect_user_demand, collect_job_statistics, collect_resource_statistics, \
-    collect_cobald_cost
+from lapis.utility.monitor import Monitoring, collect_pool_cobald_statistics, \
```

Comment:
Is there any chance of shortening these names? Just ``pool_statistics``, or perhaps a submodule as ``cobald.pool_statistics``, for example?

Reply:
I moved the functions into different modules.